pkg/provider/resource_cluster.go

```diff
@@ -403,6 +415,40 @@ func (c *clusterResource) Schema(ctx context.Context, req resource.SchemaRequest
 				Computed:            true,
 				PlanModifiers:       []planmodifier.Bool{boolplanmodifier.UseStateForUnknown()},
 			},
+			"pg_bouncer": schema.SingleNestedAttribute{
+				MarkdownDescription: "Pg bouncer.",
+				Optional:            true,
+				Computed:            true,
+				PlanModifiers: []planmodifier.Object{
+					plan_modifier.CustomPgBouncer(),
+				},
+				Attributes: map[string]schema.Attribute{
+					"is_enabled": schema.BoolAttribute{
+						MarkdownDescription: "Is pg bouncer enabled.",
+						Required:            true,
```

Comment:
should be optional as,  PgBouncer is not applicable on BAH clusters

Reply:
Maybe it has just released but I can turn on PgBouncer for BAH clusters using the UI on dev and prod. But PgBouncer here is optional, just the is_enabled field is required. There is a PgBouncer object above this field that is optional